packages/aws-client/src/S3.ts

```diff
@@ -968,7 +968,8 @@ export const multipartCopyObject = async (
     sourceObject?: AWS.S3.HeadObjectOutput,
     ACL?: AWS.S3.ObjectCannedACL,
     copyTags?: boolean,
-    copyMetadata?: boolean
+    copyMetadata?: boolean,
+    maxChunkSize?: number
```

Comment:
would it be terrible to rename the parameter to chunkSize instead of maxChunkSize?    It's technically the max size for the chunks we build but it's not the maximum possible size.

Reply:
renamed maxChunkSize to chunkSize, added docstring for chunkSize
[1cefbe8](https://github.com/nasa/cumulus/pull/2532/commits/1cefbe8da7441fb4cdf39730eff3d8c8d89dbc2f)
[5aa4517](https://github.com/nasa/cumulus/pull/2532/commits/5aa4517a8f70ea68154374cbce2e0c086a799f79)